### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by RaulTrombin**
Add readme to project, helping newcomers and included additional information about thes macro.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# cpy-rs: Python and C++ Multibinding Export Macro
+
+![cpy-rs Logo](https://github.com/patrickelectric/cpy-rs/blob/main/cpy-rs-logo.png)
```
Comment (patrickelectric) [inline README.md:3]: You forgot to add the image.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+cargo build
+```
+
+4. Your Rust functions are now accessible from Python and C++!
+
+For detailed examples and more advanced usage, please refer to the [Examples](https://github.com/patrickelectric/cpy-rs/tree/main/example) section.
+
+## Examples
+
+Here's a quick sneak peek at a Python and C++ code using a Rust function exported with `cpy-rs`:
```
Comment (joaoantoniocardoso) [inline README.md:84]: ba dum tss

File: README.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+- C++ compiler
+
+### Installation
+
+To use cpy-rs in your project, you can include it as a dependency in your Rust project's `Cargo.toml` file:
+
+```toml
+[dependencies]
+cpy-binder = "0.3.0"
+```
```
Comment (joaoantoniocardoso) [inline README.md]:
```suggestion
To use cpy-rs in your project, run `cargo add cpy-binder`, or include it as a dependency in your Rust project's `Cargo.toml` file:

```toml
[dependencies]
cpy-binder = "0.3.0"
```
```

**Review by joaoantoniocardoso — APPROVED** [review]
Cool, just one quick suggestion

File: README.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+2. Initiate your Rust module collections like following.
+
+```rust
+// Rust function using cpy macro
+export_cpy!(
+    mod my_module {
+        fn return_true() -> bool {
+            let x = true
+        }
+    }
```
Comment (patrickelectric) [inline README.md:69]: This code is not closing the scope of export_cpy

File: README.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+```
+
+2. Initiate your Rust module collections like following.
+
+```rust
+// Rust function using cpy macro
+export_cpy!(
+    mod my_module {
+        fn return_true() -> bool {
+            let x = true
```
Comment (patrickelectric) [inline README.md:67]: This code is wrong, is not returning at all

File: README.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+        }
+    }
+```
+
+3. Build your Rust project with the `cpy` feature enabled:
+
+```bash
+cargo build
```
Comment (patrickelectric) [inline README.md:75]: It's missing the build script for binding generation for C/C++

Comment (patrickelectric) [inline README.md:75]: It's missing toolchain configuration for nightly build, project do not build with current stable.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+[dependencies]
+cpy-binder = "0.3.0"
+
+### Usage
+
+1. Import the `cpy-binder` macro into your Rust project:
+
+```rust
+#[cfg(feature = "python")]
+use pyo3::{prelude::*, wrap_pyfunction};
```
Comment (patrickelectric) [inline README.md:55]: This include is not necessary.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+import my_module
+
+result = my_module.return_true()
+print(result)
+```
+
+```cpp
+// C++ code
+#include <iostream>
+#include "my_module.cpp"
```
Comment (patrickelectric) [inline README.md:97]: This include is totally wrong. And as said, it's missing binding generation for the header.

File: README.md [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+
+int main() {
+    bool result = my_rust_module::return_true();
```
Comment (patrickelectric) [inline README.md:100]: This is wrong, the cbindgen does not generate the function as namespace.

**Review by patrickelectric — CHANGES_REQUESTED** [review]
Well, I just stop the review in the middle since this PR is totally wrong. Are you using AI to generate these kind of stuff ? You are just making us lose time reviewing it.
Did you even care to try this out and review before opening the PR ?
I **truly hope** to not see that again.

**Comment by RaulTrombin** [comment]
> Well, I just stop the review in the middle since this PR is totally wrong. Are you using AI to generate these kind of stuff ? You are just making us lose time reviewing it. Did you even care to try this out and review before opening the PR ? I **truly hope** to not see that again.

I'm not.
I'm sorry to let this itens there, just thought that they could be filled after, 
The crate is missing some important topics that could be made before.
image
Was just following the rustbook [publishing](https://doc.rust-lang.org/cargo/reference/publishing.html) topic.

**Pull request closed without merging**